subxt/src/blocks/extrinsic_types.rs

```diff
@@ -558,6 +671,101 @@ impl<T: Config> ExtrinsicEvents<T> {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct ExtrinsicSignedExtensions<'a, T: Config> {
+    signed_extensions: Vec<ExtrinsicSignedExtension<'a, T>>,
+    bytes: &'a [u8],
+    ty_id: u32,
+    metadata: &'a Metadata,
+    phantom: PhantomData<T>,
+}
+
+#[derive(Debug, Clone)]
```

Comment:
I expect this can't be cloned and needs `derivative` impls (see some other structs in Subxt) to do this properly because `T != Clone`, but would be interested to see!)

Reply:
But actually, we Don't need to have the `T` in these structs at all anyway, so might all be a moot point!